src/components/tooltip/TooltipDirective.js

```diff
@@ -462,10 +527,7 @@ goog.require('ga_topic_service');
                     $http.get(htmlUrl, {
                       timeout: canceler.promise,
                       params: {
-                        lang: $translate.use(),
-                        mapExtent: mapExtent.join(','),
-                        coord: (coordinate) ? coordinate.join(',') : undefined,
-                        imageDisplay: size[0] + ',' + size[1] + ',96'
+                        lang: $translate.use()
```

Comment:
Yes. I think cadastralwebmap tooltip depends on them. There might be others as well.